lib/omb-prompt-base.sh

```diff
@@ -154,7 +154,7 @@ function scm_prompt_info {
 
 function scm_prompt_char_info {
   scm_prompt_char
-  echo -ne "${SCM_THEME_CHAR_PREFIX}${SCM_CHAR}${SCM_THEME_CHAR_SUFFIX}"
+  _omb_util_put "${SCM_THEME_CHAR_PREFIX}${SCM_CHAR}${SCM_THEME_CHAR_SUFFIX}"
```

Comment:
This changes the behavior of this function. Users may specify `\033...` in `SCM_THEME_CHAR_PREFIX`, etc.

However, this behavioral change could be harmless if all the outputs of this function are specified to `PS1` as e.g. `PS1="$(scm_prompt_char_info)"`. They become `PS1='...\033...'` after the command substitution and are finally processed within the `PS1` evaluation. If any calls are performed by `PS1='$(scm_prompt_char_info)'`, these `\033`, etc. will not be expanded and are finally included in the prompt directly. The command substitution will be expanded in the `PS1` evaluation phase, but the result will not be further evaluated.

I haven't checked whether there are places affected by this behavioral change. There doesn't seem to be any mention of this behavioral change. There doesn't seem to be an explanation about whether you have paid attention to this possible breakage. Were you aware of this possible problem? Have you confirmed all the calls of this function are not affected by this behavioral change?